packages/augur-core/source/contracts/Augur.sol

```diff
@@ -188,9 +192,9 @@ contract Augur is IAugur, IAugurCreationDataGetter {
     // Transfer
     //
 
-    function trustedTransfer(IERC20 _token, address _from, address _to, uint256 _amount) public returns (bool) {
+    function trustedTransfer(address _from, address _to, uint256 _amount) public returns (bool) {
```

Comment:
Consider naming this `trustedCashTransfer` to make it a bit more clear at the callsite what this is doing.